hoodie-client/src/test/java/com/uber/hoodie/TestHoodieClientBase.java

```diff
@@ -80,7 +83,14 @@ public void init() throws IOException {
     folder.create();
     basePath = folder.getRoot().getAbsolutePath();
     fs = FSUtils.getFs(basePath, jsc.hadoopConfiguration());
-    HoodieTestUtils.init(jsc.hadoopConfiguration(), basePath);
+    if (fs instanceof LocalFileSystem) {
```

Comment:
Ideally, if this is causing issues, we should use DFS instead of localFS like in other classes.

Reply:
There are other tests that uses localFS. We have to make a separate decision on all of these.

Reply:
I agree there are other tests that use localfs. What I'm hinting at is that if can use DFS, then may be just initialize that instead of having a conditional for local fs ?